### ISSUE TITLE
coreOS Installation USB Media boot Failed but Workstation Media booted OK

### ISSUE THREAD

**Issue by SampsonF** [coreos/fedora-coreos-tracker#724]
**Describe the bug**
I tested on three of my mahcines.

1.  Sony Vaio VGN-TZ17GN
- coreOS Media not booted at all, while Workstataion Media booted OK

2.  Desktop, Gigabyte F2A88XN-WIFI
- Workstation Media booted OK in UEFI and BIOS mode
- coreOS Media booted OK in BIOS mode, after installation, switch to UEFI boot is OK.
- coreOS Media booted to a grub shell in UEFI mode

**Reproduction steps**
Download coreOS and Workstation ISO image.  Use Fedora Media Writer to create USB boot media.

Boot Workstation USB OK.  Then try boot coreOS USB immediately at the same machine.

**Expected behavior**
coreOS USB media should boot up and allow me to Install coreOS

**Actual behavior**
At Sony notebook, coreOS USB media is not booted at all.

At Dell notebook, coreOS USB media is booted correctly

At Desktop machine, coreOS USB media failed to boot in UEFI mode.  But boot OK in BIOS mode.

**Comment by SampsonF** [comment]
dmidecode output of my Workstation machine.
Note:  Silverblue F33 media booted OK in UEFI mode also.

`# dmidecode 3.2
Getting SMBIOS data from sysfs.
SMBIOS 2.7 present.
52 structures occupying 2423 bytes.
Table at 0x000EBEE0.

Handle 0x0000, DMI type 0, 24 bytes
BIOS Information
	Vendor: American Megatrends Inc.
	Version: F6
	Release Date: 12/24/2015
	Address: 0xF0000
	Runtime Size: 64 kB
	ROM Size: 8 MB
	Characteristics:
		PCI is supported
		BIOS is upgradeable
		BIOS shadowing is allowed
		Boot from CD is supported
		Selectable boot is supported
		BIOS ROM is socketed
		EDD is supported
		5.25"/1.2 MB floppy services are supported (int 13h)
		3.5"/720 kB floppy services are supported (int 13h)
		3.5"/2.88 MB floppy services are supported (int 13h)
		Print screen service is supported (int 5h)
		8042 keyboard services are supported (int 9h)
		Serial services are supported (int 14h)
		Printer services are supported (int 17h)
		ACPI is supported
		USB legacy is supported
		BIOS boot specification is supported
		Targeted content distribution is supported
		UEFI is supported
	BIOS Revision: 4.6

Handle 0x0001, DMI type 1, 27 bytes
System Information
	Manufacturer: Gigabyte Technology Co., Ltd.
	Product Name: To be filled by O.E.M.
	Version: To be filled by O.E.M.
	Serial Number: To be filled by O.E.M.
	UUID: 03d40274-0435-051d-ed06-380700080009
	Wake-up Type: Power Switch
	SKU Number: To be filled by O.E.M.
	Family: To be filled by O.E.M.

Handle 0x0002, DMI type 2, 15 bytes
Base Board Information
	Manufacturer: Gigabyte Technology Co., Ltd.
	Product Name: F2A88XN-WIFI
	Version: x.x
	Serial Number: To be filled by O.E.M.
	Asset Tag: To be filled by O.E.M.
	Features:
		Board is a hosting board
		Board is replaceable
	Location In Chassis: To be filled by O.E.M.
	Chassis Handle: 0x0003
	Type: Motherboard
	Contained Object Handles: 0

Handle 0x0003, DMI type 3, 22 bytes
Chassis Information
	Manufacturer: Gigabyte Technology Co., Ltd.
	Type: Desktop
	Lock: Not Present
	Version: To Be Filled By O.E.M.
	Serial Number: To Be Filled By O.E.M.
	Asset Tag: To Be Filled By O.E.M.
	Boot-up State: Safe
	Power Supply State: Safe
	Thermal State: Safe
	Security Status: None
	OEM Information: 0x00000000
	Height: Unspecified
	Number Of Power Cords: 1
	Contained Elements: 0
	SKU Number: To be filled by O.E.M.

Handle 0x0004, DMI type 8, 9 bytes
Port Connector Information
	Internal Reference Designator: J1A1
	Internal Connector Type: None
	External Reference Designator: PS2Mouse
	External Connector Type: PS/2
	Port Type: Mouse Port

Handle 0x0005, DMI type 8, 9 bytes
Port Connector Information
	Internal Reference Designator: J1A1
	Internal Connector Type: None
	External Reference Designator: Keyboard
	External Connector Type: PS/2
	Port Type: Keyboard Port

Handle 0x0006, DMI type 8, 9 bytes
Port Connector Information
	Internal Reference Designator: J2A1
	Internal Connector Type: None
	External Reference Designator: TV Out
	External Connector Type: Mini Centronics Type-14
	Port Type: Other

Handle 0x0007, DMI type 8, 9 bytes
Port Connector Information
	Internal Reference Designator: J2A2A
	Internal Connector Type: None
	External Reference Designator: COM A
	External Connector Type: DB-9 male
	Port Type: Serial Port 16550A Compatible

Handle 0x0008, DMI type 8, 9 bytes
Port Connector Information
	Internal Reference Designator: J2A2B
	Internal Connector Type: None
	External Reference Designator: Video
	External Connector Type: DB-15 female
	Port Type: Video Port

Handle 0x0009, DMI type 8, 9 bytes
Port Connector Information
	Internal Reference Designator: J3A1
	Internal Connector Type: None
	External Reference Designator: USB1
	External Connector Type: Access Bus (USB)
	Port Type: USB

Handle 0x000A, DMI type 8, 9 bytes
Port Connector Information
	Internal Reference Designator: J3A1
	Internal Connector Type: None
	External Reference Designator: USB2
	External Connector Type: Access Bus (USB)
	Port Type: USB

Handle 0x000B, DMI type 8, 9 bytes
Port Connector Information
	Internal Reference Designator: J3A1
	Internal Connector Type: None
	External Reference Designator: USB3
	External Connector Type: Access Bus (USB)
	Port Type: USB

Handle 0x000C, DMI type 8, 9 bytes
Port Connector Information
	Internal Reference Designator: J9A1 - TPM HDR
	Internal Connector Type: Other
	External Reference Designator: Not Specified
	External Connector Type: None
	Port Type: Other

Handle 0x000D, DMI type 8, 9 bytes
Port Connector Information
	Internal Reference Designator: J9C1 - PCIE DOCKING CONN
	Internal Connector Type: Other
	External Reference Designator: Not Specified
	External Connector Type: None
	Port Type: Other

Handle 0x000E, DMI type 8, 9 bytes
Port Connector Information
	Internal Reference Designator: J2B3 - CPU FAN
	Internal Connector Type: Other
	External Reference Designator: Not Specified
	External Connector Type: None
	Port Type: Other

Handle 0x000F, DMI type 8, 9 bytes
Port Connector Information
	Internal Reference Designator: J6C2 - EXT HDMI
	Internal Connector Type: Other
	External Reference Designator: Not Specified
	External Connector Type: None
	Port Type: Other

Handle 0x0010, DMI type 8, 9 bytes
Port Connector Information
	Internal Reference Designator: J3C1 - GMCH FAN
	Internal Connector Type: Other
	External Reference Designator: Not Specified
	External Connector Type: None
	Port Type: Other

Handle 0x0011, DMI type 8, 9 bytes
Port Connector Information
	Internal Reference Designator: J1D1 - ITP
	Internal Connector Type: Other
	External Reference Designator: Not Specified
	External Connector Type: None
	Port Type: Other

Handle 0x0012, DMI type 8, 9 bytes
Port Connector Information
	Internal Reference Designator: J9E2 - MDC INTPSR
	Internal Connector Type: Other
	External Reference Designator: Not Specified
	External Connector Type: None
	Port Type: Other

Handle 0x0013, DMI type 8, 9 bytes
Port Connector Information
	Internal Reference Designator: J9E4 - MDC INTPSR
	Internal Connector Type: Other
	External Reference Designator: Not Specified
	External Connector Type: None
	Port Type: Other

Handle 0x0014, DMI type 8, 9 bytes
Port Connector Information
	Internal Reference Designator: J9E3 - LPC HOT DOCKING
	Internal Connector Type: Other
	External Reference Designator: Not Specified
	External Connector Type: None
	Port Type: Other

Handle 0x0015, DMI type 8, 9 bytes
Port Connector Information
	Internal Reference Designator: J9E1 - SCAN MATRIX
	Internal Connector Type: Other
	External Reference Designator: Not Specified
	External Connector Type: None
	Port Type: Other

Handle 0x0016, DMI type 8, 9 bytes
Port Connector Information
	Internal Reference Designator: J9G1 - LPC SIDE BAND
	Internal Connector Type: Other
	External Reference Designator: Not Specified
	External Connector Type: None
	Port Type: Other

Handle 0x0017, DMI type 8, 9 bytes
Port Connector Information
	Internal Reference Designator: J8F1 - UNIFIED
	Internal Connector Type: Other
	External Reference Designator: Not Specified
	External Connector Type: None
	Port Type: Other

Handle 0x0018, DMI type 8, 9 bytes
Port Connector Information
	Internal Reference Designator: J6F1 - LVDS
	Internal Connector Type: Other
	External Reference Designator: Not Specified
	External Connector Type: None
	Port Type: Other

Handle 0x0019, DMI type 8, 9 bytes
Port Connector Information
	Internal Reference Designator: J2F1 - LAI FAN
	Internal Connector Type: Other
	External Reference Designator: Not Specified
	External Connector Type: None
	Port Type: Other

Handle 0x001A, DMI type 8, 9 bytes
Port Connector Information
	Internal Reference Designator: J2G1 - GFX VID
	Internal Connector Type: Other
	External Reference Designator: Not Specified
	External Connector Type: None
	Port Type: Other

Handle 0x001B, DMI type 8, 9 bytes
Port Connector Information
	Internal Reference Designator: J1G6 - AC JACK
	Internal Connector Type: Other
	External Reference Designator: Not Specified
	External Connector Type: None
	Port Type: Other

Handle 0x001C, DMI type 9, 17 bytes
System Slot Information
	Designation: J6B2
	Type: x16 PCI Express
	Current Usage: In Use
	Length: Long
	ID: 0
	Characteristics:
		3.3 V is provided
		Opening is shared
		PME signal is supported
	Bus Address: 0000:00:02.0

Handle 0x001D, DMI type 9, 17 bytes
System Slot Information
	Designation: J6B1
	Type: x1 PCI Express
	Current Usage: In Use
	Length: Short
	ID: 1
	Characteristics:
		3.3 V is provided
		Opening is shared
		PME signal is supported
	Bus Address: 0000:00:1c.0

Handle 0x001E, DMI type 9, 17 bytes
System Slot Information
	Designation: J6D1
	Type: x8 PCI Express
	Current Usage: In Use
	Length: Short
	ID: 2
	Characteristics:
		3.3 V is provided
		Opening is shared
		PME signal is supported
	Bus Address: 0000:00:01.0

Handle 0x001F, DMI type 9, 17 bytes
System Slot Information
	Designation: J7B1
	Type: x16 PCI Express
	Current Usage: In Use
	Length: Short
	ID: 3
	Characteristics:
		3.3 V is provided
		Opening is shared
		PME signal is supported
	Bus Address: 0000:00:03.0

Handle 0x0020, DMI type 10, 6 bytes
On Board Device Information
	Type: Video
	Status: Enabled
	Description:    To Be Filled By O.E.M.

Handle 0x0021, DMI type 11, 5 bytes
OEM Strings
	String 1: To Be Filled By O.E.M.
	String 2: To Be Filled By O.E.M.
	String 3: To Be Filled By O.E.M.
	String 4: To Be Filled By O.E.M.
	String 5: To Be Filled By O.E.M.

Handle 0x0022, DMI type 12, 5 bytes
System Configuration Options
	Option 1: To Be Filled By O.E.M.

Handle 0x0023, DMI type 16, 23 bytes
Physical Memory Array
	Location: System Board Or Motherboard
	Use: System Memory
	Error Correction Type: None
	Maximum Capacity: 8 GB
	Error Information Handle: Not Provided
	Number Of Devices: 2

Handle 0x0024, DMI type 19, 31 bytes
Memory Array Mapped Address
	Starting Address: 0x00000000000
	Ending Address: 0x003FFFFFFFF
	Range Size: 16 GB
	Physical Array Handle: 0x0023
	Partition Width: 255

Handle 0x0025, DMI type 17, 34 bytes
Memory Device
	Array Handle: 0x0023
	Error Information Handle: Not Provided
	Total Width: Unknown
	Data Width: 64 bits
	Size: No Module Installed
	Form Factor: Unknown
	Set: None
	Locator: Node0_Dimm0
	Bank Locator: Node0_Bank0
	Type: Unknown
	Type Detail: None
	Speed: Unknown
	Manufacturer: Dimm0_Manufacturer
	Serial Number: Dimm0_SerNum
	Asset Tag: Dimm0_AssetTag
	Part Number: Dimm0_PartNum
	Rank: Unknown
	Configured Memory Speed: Unknown

Handle 0x0026, DMI type 126, 35 bytes
Inactive

Handle 0x0027, DMI type 17, 34 bytes
Memory Device
	Array Handle: 0x0023
	Error Information Handle: Not Provided
	Total Width: 64 bits
	Data Width: 64 bits
	Size: 8 GB
	Form Factor: DIMM
	Set: None
	Locator: Node0_Dimm1
	Bank Locator: Node0_Bank0
	Type: DDR3
	Type Detail: Synchronous Unbuffered (Unregistered)
	Speed: 1333 MT/s
	Manufacturer: Corsair           
	Serial Number: 00000000    
	Asset Tag: Dimm1_AssetTag
	Part Number: CMZ16GX3M2A18
	Rank: 2
	Configured Memory Speed: 1333 MT/s

Handle 0x0028, DMI type 20, 35 bytes
Memory Device Mapped Address
	Starting Address: 0x00000000000
	Ending Address: 0x001FFFFFFFF
	Range Size: 8 GB
	Physical Device Handle: 0x0027
	Memory Array Mapped Address Handle: 0x0024
	Partition Row Position: 1

Handle 0x0029, DMI type 17, 34 bytes
Memory Device
	Array Handle: 0x0023
	Error Information Handle: Not Provided
	Total Width: Unknown
	Data Width: 64 bits
	Size: No Module Installed
	Form Factor: Unknown
	Set: None
	Locator: Node0_Dimm2
	Bank Locator: Node0_Bank0
	Type: Unknown
	Type Detail: None
	Speed: Unknown
	Manufacturer: Dimm2_Manufacturer
	Serial Number: Dimm2_SerNum
	Asset Tag: Dimm2_AssetTag
	Part Number: Dimm2_PartNum
	Rank: Unknown
	Configured Memory Speed: Unknown

Handle 0x002A, DMI type 126, 35 bytes
Inactive

Handle 0x002B, DMI type 17, 34 bytes
Memory Device
	Array Handle: 0x0023
	Error Information Handle: Not Provided
	Total Width: 64 bits
	Data Width: 64 bits
	Size: 8 GB
	Form Factor: DIMM
	Set: None
	Locator: Node0_Dimm3
	Bank Locator: Node0_Bank0
	Type: DDR3
	Type Detail: Synchronous Unbuffered (Unregistered)
	Speed: 1333 MT/s
	Manufacturer: Corsair           
	Serial Number: 00000000    
	Asset Tag: Dimm3_AssetTag
	Part Number: CMZ16GX3M2A18
	Rank: 2
	Configured Memory Speed: 1333 MT/s

Handle 0x002C, DMI type 20, 35 bytes
Memory Device Mapped Address
	Starting Address: 0x00200000000
	Ending Address: 0x003FFFFFFFF
	Range Size: 8 GB
	Physical Device Handle: 0x002B
	Memory Array Mapped Address Handle: 0x0024
	Partition Row Position: 1

Handle 0x002D, DMI type 32, 20 bytes
System Boot Information
	Status: No errors detected

Handle 0x002E, DMI type 41, 11 bytes
Onboard Device
	Reference Designation:  Onboard LAN
	Type: Ethernet
	Status: Enabled
	Type Instance: 1
	Bus Address: 0000:00:19.0

Handle 0x002F, DMI type 7, 19 bytes
Cache Information
	Socket Designation: L1 CACHE
	Configuration: Enabled, Not Socketed, Level 1
	Operational Mode: Write Back
	Location: Internal
	Installed Size: 256 kB
	Maximum Size: 256 kB
	Supported SRAM Types:
		Pipeline Burst
	Installed SRAM Type: Pipeline Burst
	Speed: 1 ns
	Error Correction Type: Multi-bit ECC
	System Type: Unified
	Associativity: 2-way Set-associative

Handle 0x0030, DMI type 7, 19 bytes
Cache Information
	Socket Designation: L2 CACHE
	Configuration: Enabled, Not Socketed, Level 2
	Operational Mode: Write Back
	Location: Internal
	Installed Size: 4 MB
	Maximum Size: 4 MB
	Supported SRAM Types:
		Pipeline Burst
	Installed SRAM Type: Pipeline Burst
	Speed: 1 ns
	Error Correction Type: Multi-bit ECC
	System Type: Unified
	Associativity: 16-way Set-associative

Handle 0x0039, DMI type 4, 42 bytes
Processor Information
	Socket Designation: P0
	Type: Central Processor
	Family: A-Series
	Manufacturer: AuthenticAMD
	ID: 01 0F 63 00 FF FB 8B 17
	Signature: Family 21, Model 48, Stepping 1
	Flags:
		FPU (Floating-point unit on-chip)
		VME (Virtual mode extension)
		DE (Debugging extension)
		PSE (Page size extension)
		TSC (Time stamp counter)
		MSR (Model specific registers)
		PAE (Physical address extension)
		MCE (Machine check exception)
		CX8 (CMPXCHG8 instruction supported)
		APIC (On-chip APIC hardware supported)
		SEP (Fast system call)
		MTRR (Memory type range registers)
		PGE (Page global enable)
		MCA (Machine check architecture)
		CMOV (Conditional move instruction supported)
		PAT (Page attribute table)
		PSE-36 (36-bit page size extension)
		CLFSH (CLFLUSH instruction supported)
		MMX (MMX technology supported)
		FXSR (FXSAVE and FXSTOR instructions supported)
		SSE (Streaming SIMD extensions)
		SSE2 (Streaming SIMD extensions 2)
		HTT (Multi-threading)
	Version: AMD A10-7850K Radeon R7, 12 Compute Cores 4C+8G
	Voltage: 1.3 V
	External Clock: 100 MHz
	Max Speed: 3700 MHz
	Current Speed: 3700 MHz
	Status: Populated, Enabled
	Upgrade: Socket FM2
	L1 Cache Handle: 0x002F
	L2 Cache Handle: 0x0030
	L3 Cache Handle: Not Provided
	Serial Number: Not Specified
	Asset Tag: Not Specified
	Part Number: Not Specified
	Core Count: 4
	Core Enabled: 4
	Thread Count: 4
	Characteristics:
		64-bit capable

Handle 0x003A, DMI type 13, 22 bytes
BIOS Language Information
	Language Description Format: Long
	Installable Languages: 7
		en|US|iso8859-1
		de|DE|iso8859-1
		ru|RU|iso8859-5
		ko|KR|unicode
		ja|JP|unicode
		zh|CS|unicode
		zh|CT|unicode
	Currently Installed Language: en|US|iso8859-1

Handle 0x003E, DMI type 127, 4 bytes
End Of Table
`

**Comment by SampsonF** [comment]
At Sony notebook, running Workstation Live from the USB media.

Output of dmidecode

`# dmidecode
# dmidecode 3.2
Getting SMBIOS data from sysfs.
SMBIOS 2.4 present.
18 structures occupying 977 bytes.
Table at 0x000DC010.

Handle 0x0000, DMI type 0, 24 bytes
BIOS Information
	Vendor: Phoenix Technologies LTD
	Version: R0071N7
	Release Date: 07/26/2007
	Address: 0xE42C0
	Runtime Size: 113984 bytes
	ROM Size: 1 MB
	Characteristics:
		PCI is supported
		PC Card (PCMCIA) is supported
		PNP is supported
		BIOS is upgradeable
		BIOS shadowing is allowed
		ESCD support is available
		Boot from CD is supported
		Selectable boot is supported
		EDD is supported
		8042 keyboard services are supported (int 9h)
		CGA/mono video services are supported (int 10h)
		ACPI is supported
		USB legacy is supported
		AGP is supported
		IEEE 1394 boot is supported
		Smart battery is supported
		BIOS boot specification is supported
		Function key-initiated network boot is supported
		Targeted content distribution is supported
	BIOS Revision: 7.1
	Firmware Revision: 7.1

Handle 0x0001, DMI type 1, 27 bytes
System Information
	Manufacturer: Sony Corporation
	Product Name: VGN-TZ17GN_B
	Version: J002Q9HW
	Serial Number: 28203674-7005553
	UUID: 03f758c0-5b3d-11d9-8b25-0013a9ffd01c
	Wake-up Type: Power Switch
	SKU Number: N/A                             
	Family: N/A                             

Handle 0x0002, DMI type 2, 10 bytes
Base Board Information
	Manufacturer: Sony Corporation
	Product Name: VAIO                            
	Version: N/A                             
	Serial Number: N/A                             
	Asset Tag: N/A                             
	Features:
		Board is a hosting board

Handle 0x0003, DMI type 3, 17 bytes
Chassis Information
	Manufacturer: Sony Corporation
	Type: Notebook
	Lock: Not Present
	Version: N/A
	Serial Number: J002Q9HW
	Asset Tag:                                                   
	Boot-up State: Safe
	Power Supply State: Safe
	Thermal State: Safe
	Security Status: None
	OEM Information: 0x000000FF

Handle 0x0004, DMI type 4, 35 bytes
Processor Information
	Socket Designation: N/A
	Type: Central Processor
	Family: Other
	Manufacturer: GenuineIntel
	ID: F2 06 00 00 FF FB EB BF
	Signature: Type 0, Family 6, Model 15, Stepping 2
	Flags:
		FPU (Floating-point unit on-chip)
		VME (Virtual mode extension)
		DE (Debugging extension)
		PSE (Page size extension)
		TSC (Time stamp counter)
		MSR (Model specific registers)
		PAE (Physical address extension)
		MCE (Machine check exception)
		CX8 (CMPXCHG8 instruction supported)
		APIC (On-chip APIC hardware supported)
		SEP (Fast system call)
		MTRR (Memory type range registers)
		PGE (Page global enable)
		MCA (Machine check architecture)
		CMOV (Conditional move instruction supported)
		PAT (Page attribute table)
		PSE-36 (36-bit page size extension)
		CLFSH (CLFLUSH instruction supported)
		DS (Debug store)
		ACPI (ACPI supported)
		MMX (MMX technology supported)
		FXSR (FXSAVE and FXSTOR instructions supported)
		SSE (Streaming SIMD extensions)
		SSE2 (Streaming SIMD extensions 2)
		SS (Self-snoop)
		HTT (Multi-threading)
		TM (Thermal monitor supported)
		PBE (Pending break enabled)
	Version: Intel(R) Core(TM)2 CPU         U7600  @ 1.20GHz
	Voltage: 0.9 V
	External Clock: 133 MHz
	Max Speed: 1200 MHz
	Current Speed: 1200 MHz
	Status: Populated, Enabled
	Upgrade: None
	L1 Cache Handle: 0x0005
	L2 Cache Handle: 0x0006
	L3 Cache Handle: 0x0007
	Serial Number: N/A
	Asset Tag: N/A
	Part Number: N/A

Handle 0x0005, DMI type 7, 19 bytes
Cache Information
	Socket Designation: L1 Cache
	Configuration: Enabled, Socketed, Level 1
	Operational Mode: Write Back
	Location: Internal
	Installed Size: 32 kB
	Maximum Size: 32 kB
	Supported SRAM Types:
		Burst
		Pipeline Burst
		Asynchronous
	Installed SRAM Type: Asynchronous
	Speed: Unknown
	Error Correction Type: None
	System Type: Other
	Associativity: 8-way Set-associative

Handle 0x0006, DMI type 7, 19 bytes
Cache Information
	Socket Designation: L2 Cache
	Configuration: Enabled, Socketed, Level 2
	Operational Mode: Write Back
	Location: Internal
	Installed Size: 2 MB
	Maximum Size: 2 MB
	Supported SRAM Types:
		Burst
		Pipeline Burst
		Asynchronous
	Installed SRAM Type: Burst
	Speed: Unknown
	Error Correction Type: None
	System Type: Data
	Associativity: 8-way Set-associative

Handle 0x0007, DMI type 7, 19 bytes
Cache Information
	Socket Designation: L3 Cache
	Configuration: Disabled, Not Socketed, Level 3
	Operational Mode: Unknown
	Location: Unknown
	Installed Size: 0 kB
	Maximum Size: 0 kB
	Supported SRAM Types:
		Burst
		Pipeline Burst
		Asynchronous
	Installed SRAM Type: Burst
	Speed: Unknown
	Error Correction Type: None
	System Type: Data
	Associativity: 4-way Set-associative

Handle 0x0009, DMI type 11, 5 bytes
OEM Strings
	String 1: JPBL-003668
	String 2: FNC-CCIA0eeeHDPW
	String 3: 4L9P00000000fe850b668548e7ff
	String 4: Reserved              
	String 5: Reserved              

Handle 0x000A, DMI type 16, 15 bytes
Physical Memory Array
	Location: System Board Or Motherboard
	Use: System Memory
	Error Correction Type: Unknown
	Maximum Capacity: Unknown
	Error Information Handle: Not Provided
	Number Of Devices: 2

Handle 0x000B, DMI type 17, 27 bytes
Memory Device
	Array Handle: 0x000A
	Error Information Handle: Not Provided
	Total Width: 32 bits
	Data Width: 32 bits
	Size: 2 GB
	Form Factor: SODIMM
	Set: None
	Locator: SODIMM1
	Bank Locator: Bank 0
	Type: DDR
	Type Detail: Unknown
	Speed: Unknown
	Manufacturer: Not Specified
	Serial Number: Not Specified
	Asset Tag: Not Specified
	Part Number: Not Specified

Handle 0x000C, DMI type 17, 27 bytes
Memory Device
	Array Handle: 0x000A
	Error Information Handle: Not Provided
	Total Width: Unknown
	Data Width: Unknown
	Size: No Module Installed
	Form Factor: SODIMM
	Set: None
	Locator: SODIMM2
	Bank Locator: Bank 1
	Type: Unknown
	Type Detail: Unknown
	Speed: Unknown
	Manufacturer: Not Specified
	Serial Number: Not Specified
	Asset Tag: Not Specified
	Part Number: Not Specified

Handle 0x000D, DMI type 19, 15 bytes
Memory Array Mapped Address
	Starting Address: 0x00000000000
	Ending Address: 0x0007FFFFFFF
	Range Size: 2 GB
	Physical Array Handle: 0x000A
	Partition Width: 2

Handle 0x000E, DMI type 20, 19 bytes
Memory Device Mapped Address
	Starting Address: 0x00000000000
	Ending Address: 0x0007FFFFFFF
	Range Size: 2 GB
	Physical Device Handle: 0x000B
	Memory Array Mapped Address Handle: 0x000D
	Partition Row Position: Unknown
	Interleave Position: Unknown
	Interleaved Data Depth: Unknown

Handle 0x000F, DMI type 20, 19 bytes
Memory Device Mapped Address
	Starting Address: 0x0007FFFFC00
	Ending Address: 0x0007FFFFFFF
	Range Size: 1 kB
	Physical Device Handle: 0x000C
	Memory Array Mapped Address Handle: 0x000D
	Partition Row Position: Unknown
	Interleave Position: Unknown
	Interleaved Data Depth: Unknown

Handle 0x0010, DMI type 32, 11 bytes
System Boot Information
	Status: No errors detected

Handle 0x0011, DMI type 136, 6 bytes
OEM-specific Type
	Header and Data:
		88 06 11 00 5A 5A

Handle 0x0012, DMI type 127, 4 bytes
End Of Table
`

**Comment by jlebon** [comment]
As a sanity-check, instead of using Fedora Media Writer for the Fedora CoreOS live ISO, can you try using `dd` as detailed here: https://docs.fedoraproject.org/en-US/quick-docs/creating-and-using-a-live-installation-image/index.html#_using_a_direct_write_method ?

**Comment by SampsonF** [comment]
Testing the Sony notebook
1.  Enter BIOS, put all USB boot options (USB Floppy, USB CD, USB Storage) at higher priority than Internal HDD
2. Boot with Fedora Workstation Media - got the "Test / Install" screen as expected.
3. Insert coreOS USB Media, reboot.
- Result - grub menu of the Internal HDD appears.  coreOS Media is not booted at all.

ISO Source:
https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/33.20210104.3.0/x86_64/fedora-coreos-33.20210104.3.0-live.x86_64.iso  

USB media creation method:
# dd if=fedora-coreos-33.20210104.3.0-live.x86_64.iso  of=/dev/sde bs=8M status=progress oflag=direct
789577728 bytes (790 MB, 753 MiB) copied, 124 s, 6.4 MB/s
94+1 records in
94+1 records out
789577728 bytes (790 MB, 753 MiB) copied, 124.079 s, 6.4 MB/s

**Comment by gdonval** [comment]
@jlebon I have the same problem (and a partial solution!) on a (venerable) Asus CS-B motherboard.

For some reason some bioses get confused with the MBR+EFI partitioning scheme: `grub` is thinking its root is in `(hd0, msdos2)` (on my computer, evidently) but `(hd0, msdos2)` is the `isolinux` root (again, YMMV). I could get coreos to boot by first loading grubs config file (from what I recall):

```
grub >  configfile (hd0)/EFI/fedora/grub.cfg
```

then press `e` to edit grub's entry and prepend all paths (1 linux + 2 initramfs images) with grub's partition (`(hd0)` in my case). I suppose one could just do the following instead (not tested):

```
grub >  set root=(hd0)
grub >  configfile (hd0)/EFI/fedora/grub.cfg
```

In short, grub does not find the correct root on those motherboards and instead uses isolinux's and fails.

**Comment by gdonval** [comment]
Had to redeploy today:

```
grub >  set root=(hd0)
grub >  configfile /EFI/fedora/grub.cfg
```

works! I don't know how to fix the problem on the actual ISO though.

**Comment by bgilbert** [comment]
Thanks for the reports, everyone.  The ISO dropping to a grub shell should be fixed by https://github.com/coreos/coreos-assembler/pull/2404.  That should address everything described here except for the problem with @SampsonF's Vaio.  I'm going to mark 2404 as fixing this bug anyway, so we don't accumulate multiple issues in a single bug report.

I'll post here once there's a nightly build with the fix for testing.  @SampsonF, would you be willing to try that build with your Vaio and file a new bug if it still doesn't work?

**Comment by SampsonF** [comment]
Sure.  I will try once the build is available.

**Comment by miabbott** [comment]
Folks may try a test ISO from here - https://miabbott.fedorapeople.org/fedora-coreos-34.20210902.dev.0-live.x86_64.iso

**Comment by SampsonF** [comment]
> Folks may try a test ISO from here - https://miabbott.fedorapeople.org/fedora-coreos-34.20210902.dev.0-live.x86_64.iso
> 
> Context - [coreos/coreos-assembler#2404 (review)](https://github.com/coreos/coreos-assembler/pull/2404#pullrequestreview-745051698)

This ISO boots OK to the coreOS live environment for all the three machines.

Thank you very much!

**Comment by dustymabe** [comment]
The fix for this went into stable stream release `34.20210821.3.0`.
